src/components/TodoItem/TodoItem.jsx

```diff
@@ -0,0 +1,57 @@
+import PropTypes from 'prop-types'
+import { CheckIcon, CloseIcon } from '../../assets/svgs'
+import styles from './TodoItem.module.scss'
+import { classNames } from '../../styles'
+
+const cx = classNames.bind(styles)
+
+function TodoItem({ todo, setTodoList }) {
+  const handleChange = (e) => {
+    const { dataset, checked } = e.currentTarget
+    const { id } = dataset
```

Comment:
```suggestion
    const { dataset: { id }, checked } = e.currentTarget
```
이렇게 한 줄로 바꾸는 건 어떨까요? `dataset`은 이후에 쓰지 않는 값이라 여기서 destructing 해줘도 될 것 같아서요
개인 취향 차이일 것 같은데 어떤게 더 가독성이 좋을지 의견 부탁드립니다ㅎㅎ